### PR TITLE
add existdb

### PR DESCRIPTION
**Pull request by marcelotrevisani**

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
@ocefpaf could you please help me here?
Trying to create a recipe for this java project

**Comment by ocefpaf** [comment]
> @ocefpaf could you please help me here? Trying to create a recipe for this java project.

I don't know the first thing about Java! Looks OK to me though.

Ping @conda-forge/staged-recipes

**Comment by CJ-Wright** [comment]
@conda-forge/help-java

**Comment by marcelotrevisani** [comment]
@conda-forge-admin , please lint

File: recipes/existdb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - openjdk
+  run:
+    - openjdk
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/run.sh          # [unix]
+    - test -f ${PREFIX}/bin/shutdown.sh     # [unix]
+    - test -f ${PREFIX}/bin/run.bat         # [win]
+    - test -f ${PREFIX}/bin/shutdown.bat    # [win]
```
Comment (scopatz) [inline recipes/existdb/meta.yaml]: Wow! Those are rather general names!

Comment (marcelotrevisani) [reply recipes/existdb/meta.yaml]: I totally agree, maybe would be good to create a folder inside of the bin

Comment (scopatz) [reply recipes/existdb/meta.yaml]: I think that is up to you / the users.

**Comment by marcelotrevisani** [comment]
I think that's fine now. I'm going to merge this

**Pull request merged**